conversion.md

```diff
@@ -128,3 +128,8 @@ If there is a conflict (same key but different value) between an implicit annota
 
 A converter MAY add annotations which have keys not specified in the image.
 A converter MUST NOT modify the values of annotations specified in the image.
+
+Note there is a risk that some annotations might be used by container runtimes to do operations that pose a security risk (such as running container hooks on the host or modifying security-related aspects of the container configuration).
+As with any runtime specification configuration, generators SHOULD verify that the generated container configuration is safe before it is used to create a container.
+A converter MAY choose to not include annotations specified in the image, if the annotations are considered to be unsafe by the converter's security policy.
```

Comment:
```suggestion
A converter MAY choose to not include annotations specified in the image, if the annotations are considered to be unsafe by the converter's security policy.

**Implementor's Note:** Docker/Moby and similar popular implementations are known to exclude all annotations specified in the image.
```

Reply:
Does Docker actually claim to support OCI image-spec in any significant way? https://github.com/moby/moby/issues/25779 has still not been resolved.

The reference implementation of the image-spec (umoci) does implement this behaviour. I would prefer to not undercut the SHOULD and MUST portions of this specification by saying "Docker doesn't support this" when in fact Docker doesn't really support the rest of this spec in the first place...

Reply:
I've added a note, but I mention that copying Docker's example is not recommended.